Frameworks/CoreGraphics/DWriteWrapper.mm

```diff
@@ -97,8 +97,9 @@ HRESULT UpdateCollection(const ComPtr<IDWriteFactory>& dwriteFactory, const ComP
 }
 
 // Private helper, wraps around GetUserDefaultLocaleName() and returns a default of "en-us" if it fails
-static inline const wchar_t* __GetUserDefaultLocaleName(wchar_t* buf, size_t bufferSize) {
-    int defaultLocaleSuccess = GetUserDefaultLocaleName(buf, bufferSize);
+template <size_t size>
```

Comment:
I don't see any issue with that.  It is standard practice to pass a buffer and a length.  Now you are constrained to passing an array instead.